Formula/mesa.rb

```diff
@@ -23,13 +23,14 @@ class Mesa < Formula
   depends_on "linuxbrew/xorg/libvdpau" => :build
   depends_on "linuxbrew/xorg/wayland-protocols" => [:recommended, :build]
   depends_on "linuxbrew/xorg/xorgproto" => :build
-  depends_on "llvm@4" => :build
+  depends_on "llvm@7" => :build
```

Comment:
It reminds me that I was waiting on `llvm@4` formula to get a Linuxbrew bottle. Initially, I too was [using](https://github.com/Linuxbrew/homebrew-xorg/blame/4cf6cf38bd369d299b44602e932f9a8623286ae3/Formula/mesa.rb) `llvm` but then [switched](https://github.com/Linuxbrew/homebrew-xorg/commit/4a5530d25a150028648d76e730988c8574ed6790) to `llvm@4` because `llvm` was [causing](https://github.com/Linuxbrew/homebrew-xorg/pull/375) some trouble. As you can see (if you have a look at the last link), we had a plan to stop using `llvm` even then.

Reply:
Yup, I've read all of that thread before submitting this. The issues with `llvm` are apparently not the same as before in mesa 19, but rather due to some (widely known) breaking changes in LLVM 8, which is why the switch to `llvm@7` worked out well.